docs/user-guide/zowe-getting-started-tutorial.md

```diff
@@ -11,14 +11,14 @@ Learn how to start using Zowe components, applications, and plug-ins.
 ## Zowe client-side components
 
 - [Using Zowe CLI](cli-using-usingcli.md)
-- [Using Zowe Explorer](ze-usage.md)
+- [Using Zowe Explorer for Visual Studio Code](ze-usage.md)
+- [Using Zowe Explorer for IntelliJ](intellij-using.md)
```

Comment:
I would put the ZE for IntelliJ link above the ZE for VS Code link (for alphabetic order).